tests/test_space.cairo

```diff
@@ -7,8 +7,48 @@ from starkware.cairo.common.bool import TRUE, FALSE
 from starkware.starknet.common.syscalls import get_contract_address, get_caller_address
 from protostar.asserts import assert_eq
 
+# @external
```

Comment:
why commented?

Reply:
It doesn't pass since a few push. Even on master

Reply:
Will fix later. In other PR